package.json

```diff
@@ -14,7 +14,8 @@
   "scripts": {
     "prebuild": "run-s test",
     "build": "microbundle-crl --no-compress --format modern,cjs --css-modules false",
-    "start": "echo Consider running: yarn sync-to-ui ; echo ; microbundle-crl watch --no-compress --format modern,cjs --css-modules false",
+    "start": "microbundle-crl watch --no-compress --format modern,cjs --css-modules false",
+    "concurrently": "./scripts/concurrently.sh",
```

Comment:
I would rename it to `start-asisted-ui` to be clear about consequences. We are developing even the `uhc-portal` exclusively, without `assisted-ui` involved.

Reply:
@mareklibra I'm not sure I understand what you mean by "We are developing even the uhc-portal exclusively, without assisted-ui involved" (What do you call `uhc-portal`?). I named it this way following @jtomasek recommendation. Personally I would rather name it `start:dev` just to keep it short and also to reflect the fact that the script is modifying the way it launches the project. Anyway, it can be whatever you decide :)

Reply:
When developing the `assisted-ui-lib`, I recently choose from following options based on actual context:
- see changes in assisted-ui (this is what this PR recently addresses)
- see changes in uhc-portal only (I do not care about assisted-ui and don't want it to be running)
- see changes in both

Since uhc-portal dev env execution brings another level of complexity, I am fine in simplifying just the `assisted-ui` flow now. My suggestion about renaming the yarn script is motivated by keeping other flows seamlessly in the play.

Reply:
Now I understand your point.

Honestly I couldn't adapt this solution for the `uhc-portal` since I'm not familiar (yet?) with that project/codebase. I'll be more than happy to make such adaptation if you'll find it useful though.

If I rename the script to `start-assisted-ui` will that be enough for now?

Reply:
yes :-)